demo/src/Components/Pagination/index.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+
+const Pagination = () => (
+  <section className="o-section-md">
+    <div className="o-site-wrap o-site-wrap--padding">
+      <h4>Pagination</h4>
+      <nav className="c-pagination" aria-label="Page navigation">
```

Comment:
I don't see any styling for `pagination` so can remove the classame off this and for the others call them `pagination...` rather than `pagination-list...`

Reply:
done